tests/KMS.test.ts

```diff
@@ -170,6 +170,43 @@ test(
   },
 )
 
+test(
+  "KMS Locate",
+  async () => {
+    const client = new KmsClient(
+      `http://${process.env.KMS_HOST || "localhost"}:9998`,
+    )
+    if (!(await client.up())) {
+      console.error("No KMIP server. Skipping test")
+      return
+    }
+    const TAG = (Math.random() * 100000000).toString()
+    const uniqueIdentifier = await client.createSymmetricKey(
+      SymmetricKeyAlgorithm.AES,
+      256,
+      undefined,
+      [TAG],
+    )
+    const uniqueIdentifier2 = await client.createSymmetricKey(
+      SymmetricKeyAlgorithm.AES,
+      256,
+      undefined,
+      [TAG],
+    )
+
+    const uniqueIdentifiers = await client.getUniqueIdentifiersByTags([TAG])
```

Comment:
it could be interesting to retrieve objects of different types (like `SymmetricKey` and `PublicKey`).
The KMIP documentation (https://docs.oasis-open.org/kmip/kmip-spec/v2.0/os/kmip-spec-v2.0-os.html#_Toc6497553) allow this.
```
Note: the Attributes structure MAY be empty indicating all objects should match.
```